docs/pages/ktable.vue

```diff
@@ -210,8 +232,141 @@
       </DocsShow>
       <!-- eslint-enable -->
 
-    </DocsPageSection>
+      <!--Table with Default Sort-->
+      <h3>Table with Default Sort</h3>
+      <p>
+        This is an example to show how <code>KTable</code> can be used with the <code>defaultSort</code> attribute to sort the table based on a particular column. This is useful if the table needs to be sorted based on any column from the beginning itself (by default). The <code>defaultSort</code> attribute can be used irrespective of the <code>sortable</code> attribute (as <code>sortable</code> is used to configure whether the client has sorting capabilities or not). 
```

Comment:
Few tweaks that may help with clarity and conciseness

```suggestion
        This is an example to show how to use the <code>defaultSort</code> prop to sort the table based on a particular column upon the initial load. The <code>defaultSort</code> attribute can be used irrespective of the <code>sortable</code> attribute.
```

Reply:
> can be used irrespective of the <code>sortable</code> attribute

Relatedly, I wonder if bumping this whole section up on the documentation page, right between non-sortable and sortable examples, would help understanding that this applies in both modes.

Reason being that the documentation flow overall goes from non-sortable => sortable, increasingly revealing more complex scenarios.

Reply:
My two cents on the same would be that it's better to have it after the introduction of the `sortable` attribute so that when someone is reading the documentation for the first time, he is first informed of both the "base" variants of the table (sortable and un-sortable), and then is informed about the additional functionalities and APIs that the component has. 

Having it placed between the non-sortable and sortable examples might feel like a jump, as the user isn't aware of the functionality of the `sortable` attribute until after he reads the `sortable` section. 

However, please let me know if you disagree on the same!

Reply:
I think you've got a good point, @EshaanAgg. Let's leave as is.